main.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+from datetime import datetime
+
+import WeatherManComputations
+
+OKBLUE = '\033[34m'
+OKRED = '\033[31m'
+Default = "\033[39m"
+
+
+def problem_1(year_data):
+    max_temp_data = max(year_data, key=lambda x: int(x.max_temp) if x.max_temp else -200)
+    min_temp_data = min(year_data, key=lambda x: int(x.min_temp) if x.min_temp else 200)
+    max_humid = max(year_data, key=lambda x: int(x.max_humidity) if x.max_humidity else 0)
+    print("Highest: ",
+          max_temp_data.max_temp + " C on " + datetime.strptime(max_temp_data.pkt, '%Y-%m-%d').strftime("%B %d"))
+    print("Lowest: ",
+          min_temp_data.min_temp + " C on " + datetime.strptime(min_temp_data.pkt, '%Y-%m-%d').strftime("%B %d"))
+    print("Humid: ", max_humid.max_humidity + " % on " + datetime.strptime(max_humid.pkt, '%Y-%m-%d').strftime("%B %d"))
+
+
+def problem_2(year_data):
+    max_avg_temp = max(year_data, key=lambda x: int(x.mean_temp) if x.mean_temp else -200)
+    min_avg_temp = min(year_data, key=lambda x: int(x.mean_temp) if x.mean_temp else 200)
+    print("Highest Average: " + max_avg_temp.mean_temp + " C")
+    print("Min Average: " + min_avg_temp.mean_temp + " C")
+    print("Average Mean Humidity: " + str(average_humidity(year_data)) + " %")
+
+
```

Comment:
Let's Move this functionality into another class Suggestion : WeatherReporter. which will hold all the loaded data returned from Weather Data Loader  Module and uses that data to calculate and show the output.